### ISSUE TITLE
Kiwix-serve should automatically open one ZIM file 

### ISSUE THREAD

**Issue by kelson42** [kiwix/kiwix-tools#53]
_From @kelson42 on August 31, 2016 18:46_

... and avoid displaying the catalog page if only one ZIM file.

_Copied from original issue: kiwix/kiwix-xulrunner#299_

**Comment by kelson42** [comment]
Not sure anymore this is a good idea. Closing it for now

**Comment by kelson42** [comment]
In fact IMO we need that for KaiOS and we probably also need to remove the human_id part of the URL in case of only one ZIM file.

**Comment by mgautierfr** [comment]
>  we probably also need to remove the human_id part of the URL in case of only one ZIM file

I'm not sure about this. We tell the browser to cache indefinitely the content we send. This work well because we have a "unique" content per url.
If we remove the human_id from the url, we may end with different contents (because different zims) using the same url.

Saying that, we have the same issue with the nodatealias feature of kiwix-serve.

**Comment by kelson42** [comment]
@mgautierfr yes, not sure this is a problem or not. You see the problem here https://en.wikimed.kaios.kiwix.org
